### ISSUE TITLE
BUG: The website says that 1.24.0 was released when in fact 1.24.1 is the latest version

### ISSUE THREAD

**Issue by yurivict** [numpy/numpy.org#624]
### Describe the issue:

[website](https://numpy.org/)

### Reproduce the code example:

```python
n/a
```

### Runtime information:

n/a

**Comment by stefanv** [comment]
1.24.3, I believe. I mentioned this to @charris, but that's what was intended apparently.

**Comment by InessaPawson** [comment]
@stefanv @yurivict Indeed, we decided not to feature minor maintenance releases on the homepage. However, I see how the announcement about NumPy 1.24.0 in the news bar can be misleading for our users. I’ll submit a PR to add the news about the NumPy docs team leadership transition instead.
